package.json

```diff
@@ -8,7 +8,7 @@
 	"dependencies": {
 		"@types/node": "^14.14.32",
 		"@types/uuid": "^8.3.0",
-		"ardrive-core-js": "0.4.0",
+		"ardrive-core-js": "latest",
```

Comment:
This seems presumptive that we'll always be compatible with the latest version. Could we just peg to the best known version for now?